app/_data/docs_nav_gateway_2.5.x.yml

```
@@ -204,7 +173,7 @@
         - text: Securing the Admin API
           url: /admin-api/secure-admin-api
     - text: Plugin Development Kit
-      url: /pdk
+      url: /pdk-reference
       items:
         - text: kong.client
           url: /pdk/kong.client
```

Need to add /pdk-reference/ to all of these module links, otherwise they're going to be broken.

```suggestion
          url: /pdk-reference/pdk/kong.client
```